install.html

```diff
@@ -299,7 +294,7 @@ <h1 id="install"><i class="fa fa-wrench"></i>Installing the Hypervisor</h1>
                                 <li>Navigate to the <a target="_blank" href="https://www.virtualbox.org/wiki/Linux_Downloads">Linux specific download page</a>.</li>
                                 <li>Follow the instructions therein to install.</li>
                                 <li>Navigate to <a target="_blank" href="https://www.virtualbox.org/wiki/Downloads">generic download page</a> and download VirtualBox Extension Pack</li>
-                                <li>TBA (install extension pack)</li>
+                                <li>After downloading, double-click on the Extension Pack to install.</li>
```

Comment:
Are you sure that double-clicking the extension pack works in a **LInux** OS? If you really are, then this is OK.

Reply:
Replaced this with instructions on how to browse for the exp pack from virtualbox